Sources/Latte+UniversalLink.swift

```diff
@@ -0,0 +1,72 @@
+import Foundation
+import UIKit
+
+@available(iOS 13.0, *)
+public protocol LatteLinkHandler {
+    func handle(
+        context: UINavigationController,
+        authgear: Authgear,
+        handler: @escaping Latte.ResultHandler<Void>
+    )
+}
+
+@available(iOS 13.0, *)
+public extension Latte {
+    struct ResetLinkHandler: LatteLinkHandler {
+        let customUIEndpoint: String
+        let query: [URLQueryItem]?
+
+        public func handle(
+            context: UINavigationController,
+            authgear: Authgear,
+            handler: @escaping Latte.ResultHandler<Void>
+        ) {
```

Comment:
Should move the main logic to `Latte+Flows`, to consolidate UI related flows into single place.